utilities/xmlrpcpp/src/XmlRpcValue.cpp

```diff
@@ -610,16 +611,52 @@ namespace XmlRpc {
       default:           break;
       case TypeBoolean:  os << _value.asBool; break;
       case TypeInt:      os << _value.asInt; break;
-      case TypeDouble:   os << _value.asDouble; break;
+      case TypeDouble:
+        {
+          static std::once_flag once;
+          int ret, required_size;
```

Comment:
A single `int` variable (`required_size`) should be sufficient. It can also be declared at the same time when being defined.